tests/net/socket/poll/prj.conf

```diff
@@ -20,7 +20,7 @@ CONFIG_NET_CONFIG_MY_IPV6_ADDR="2001:db8::1"
 CONFIG_NET_CONFIG_NEED_IPV6=y
 
 CONFIG_MAIN_STACK_SIZE=2048
-CONFIG_ZTEST_STACKSIZE=1280
+CONFIG_ZTEST_STACKSIZE=2048
```

Comment:
Instead of doing that, can you instead modify the EXTRA_STACK_SIZE for the test?

Reply:
@ioannisg Could you elaborate? I don't know what EXTRA_STACK_SIZE is, and there doesn't seem to be any occurrence of this symbol in the repository (with the exception to some old release notes).

Reply:
Yeap, take a look here:
```
K_THREAD_STACK_DEFINE(ztest_thread_stack, CONFIG_ZTEST_STACKSIZE +
		      CONFIG_TEST_EXTRA_STACKSIZE);
```

config TEST_EXTRA_STACKSIZE is used heavily in our test suite, instead of modifying the ZTEST Stack size.

Reply:
Thanks for clarifying, I'll handle the issue with `TEST_EXTRA_STACKSIZE` then. Let's drop the commit that modifies the stack size here.